app.py

```diff
@@ -121,11 +121,24 @@ def compute_sample_sizes(round_contest):
         round_contest.sample_size = sample_size_90        
         sample_ballots(election, the_round)
 
+    # update the parent round's sample size options
+    if the_round.sample_size_options is None:
+        the_round.sample_size_options = round_contest.sample_size_options
+
+    else:
+        round_options = json.loads(the_round.sample_size_options)
+
+        # TODO: this assumes that the lists are always in the same order
+        for i, option in enumerate(sample_size_options):
```

Comment:
At one point this was a monster single-line lambda expression but it was so ugly that I just went with a simple loop instead. Could revert to a one-liner to "flex" if someone finds it entertaining. ;)

Reply:
Relying on the list order is not great, couldn't find a good way to "id" the options (some have types, some don't). Probably an easy fix, didn't have the time to figure out the most elegant way.